services/apps/alcs/src/portal/application-submission/application-owner/application-owner.controller.ts

```diff
@@ -159,19 +159,23 @@ export class ApplicationOwnerController {
     //Create Owner
     if (!data.ownerUuid) {
       await this.ownerService.deleteNonParcelOwners(applicationSubmission.uuid);
-      const newOwner = await this.ownerService.create(
-        {
-          email: data.email,
-          typeCode: data.type,
-          lastName: data.lastName,
-          firstName: data.firstName,
-          phoneNumber: data.phoneNumber,
-          organizationName: data.organization,
-          applicationSubmissionUuid: data.applicationSubmissionUuid,
-          crownLandOwnerType: data.crownLandOwnerType,
-        },
-        applicationSubmission,
-      );
+      const newOwner =
```

Comment:
Previously this check wasn't needed as when selecting something other than agent or government we always passed the ownerUuid. Why has this changed?

Reply:
The compromise we decided on necessitated being able to unset the primary contact. This was impossible previously. This change makes it so that when selecting any other type, if UUID is set to `undefined`, it sets the primary contact to `undefined` for the submissions.

Reply:
Got it, Thanks!